RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5958,3 +5958,69 @@ int meatReserve()
 	
 	return reserve_gnasir + reserve_diary + reserve_island + reserve_extra;
 }
+
+// This function returns the number of bander runaways that will be available after maximizing for fam weight
+int auto_banderRunsAvailable() {
+	if (in_quantumTerrarium()) {
+		if ((my_familiar() == $familiar[Frumious Bandersnatch] && have_skill($skill[The Ode to Booze])) || my_familiar() == $familiar[Pair of Stomping Boots]) {
+			maximize("familiar weight, -tie", false); // Need to maximize or simulate maximizing in accordance with standard autoscend procedure
+			return (((familiar_weight (my_familiar()) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int());
+		}
+		else {
+			return 0;
+		}
+	}
+	else if (have_familiar($familiar[Frumious Bandersnatch]) && have_skill($skill[The Ode to Booze])) {
+		use_familiar($familiar[Frumious Bandersnatch]);
```

Comment:
we don't change familiars until we adventure unless there's absolutely no other choice otherwise it'll be flipping back and forth all over the place causing server hits. `handleFamiliar($familiar)` sets the familiar for the next adventure (see auto_familiar.ash)